services/can_logger/can_logger.py

```diff
@@ -210,6 +254,8 @@ def log_can(can_interface):
 host_port = os.environ['socketcand_port']
 host_interfaces = os.environ['can_interface']
 log_to = os.environ['log']
+frequency = int(os.environ['frequency'])
+interval = os.environ['rotation_interval']
```

Comment:
So if we wanted to rotate logs every 4 hours, `interval` would be `'h'` and `frequency` would be `4`? or would that mean we rotate 4 times an hour?  We should clarify these environmental variable names

Reply:
Good point. Actually it means we are rotating every 4 hours. For this case, `period` sounds much better.